src/components/ExtractorResponseEditor/ExtractorResponseEditor.tsx

```diff
@@ -422,16 +422,40 @@ class ExtractorResponseEditor extends React.Component<Props, State> {
         document.removeEventListener("Test_SelectWord", this.onTestSelectWord)
     }
 
-    // For end 2 end unit testing.
+    /**
+     * For cypress end to end unit testing.  Event selects phrase in entity labeller
+     * The following will select the word "hello" in the second row (row is 1 based)
+     *   var event = new CustomEvent("Test_SelectWord", { detail: { phrase: "hello", r:ow: 2 }})
```

Comment:
typo `row: 2`, would also consider the word `index` to be more precise and dropping the `-1` logic. This is for test code not for end users so it doesn't help

Reply:
I called it row rather then index as it is 1 based

Reply:
Yea, i noticed and i'm saying since people don't know how you implemented it row as 0 based or row as 1 based, it was removing ambiguity of calling it index. Anyways doesn't matter

Reply:
I'd prefer it to be zero based since only a programmer is going to use it and that is what we expect. And yes I noticed you called it row, but it could also be called rowIndex if you really think that makes a difference.

Reply:
Looks like there is a typo in the comment:
r : ow: 2
The extra colon.

Reply:
Ok - will switched to zero based